common/src/main/java/com/microsoft/identity/common/internal/cache/CacheKeyValueDelegate.java

```diff
@@ -72,6 +73,7 @@
      */
     public static final String CACHE_VALUE_SEPARATOR = "-";
     private static final String FOCI_PREFIX = "foci-";
+    private static final String REVERSED_PRT_CLIENT_ID = "1";
```

Comment:
What is meant here by "reversed"? This naming unclear to me

Reply:
It's typo. Updated to RESERVED_PRT_CLIENT_ID.
Just like [windows code](https://github.com/AzureAD/microsoft-authentication-library-for-cpp/blob/fdf896d7d333c6a896f3a5ec5f656e2ee0429e8a/source/windows/storage/StorageWorker.hpp#L92 )

Reply:
Should we use the actual clientId that you're using for PRT acquisition?

Reply:
> Should we use the actual clientId that you're using for PRT acquisition?

Same question

Reply:
We can change it, but then we'd have to update code on other platforms too. I don't know if we want to make that step before we have to.

Reply:
(It looks like this is by-design: is it documented though?)

Reply:
I have no idea. It's same as windows code. Let me know your decision.

Reply:
Gotcha, thanks @richard1122 

@oldalton What's your take on this? This seems to run counter to the schema's definition of `client_id` with respect to `PrimaryRefreshTokens`. Personally, I'm not sure I'd have done it this way: I may have used the caller's real client_id and then had lookup logic that can handle either the id _or_ a wildcarded value (meaning "any")

/cc @jbzdarkid Who would have an interest in this

Reply:
I agree - we use real broker clientId in iOS and macOS broker as well for PRT storage, so this is inconsistent with it. However, if it would be a breaking SSO change and it doesn't conflict with other PRT storages, then agree with @jbzdarkid that we might not want to do it. If it's not a breaking SSO change though, let's make the change now rather than after we ship.

Reply:
OK, so we are going to:
- Save real client id in cache key
- Fetch by real client id or (wildcard -- any), caller decide

I'll make the change and update tests.

Reply:
Update, please help review, thanks !